src/app/tasks/services/tasks.service/index.ts

```diff
@@ -83,6 +83,7 @@ class TaskManagerService {
     return {
       taskId: task.id,
       status: task.status,
+      item: task?.file || task?.folder,
```

Comment:
why the task is maybe undefined if `task.id` and `task.status` are not?

Reply:
It has no sense. I remove it